helm-chart/renku/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
-appVersion: '1.0'
+appVersion: "1.0"
 description: A Helm chart for the Renku platform
 name: renku
 icon: https://github.com/SwissDataScienceCenter/renku-sphinx-theme/raw/master/renku_sphinx_theme/static/favicon.png
-version: 0.18.1-2.01781e6
+version: 0.25.6
```

```suggestion
version: 0.0.1-set.by.chartpress
```

If we set this to a different specific value then we have to keep updating it on every commit. It is easier to have it like this. We used to have it set to this a few months ago. I suspect a specific version got accidentally committed and merged.